packages/components/pie-button/rollup.config.js

```diff
@@ -0,0 +1,22 @@
+import resolve from '@rollup/plugin-node-resolve';
```

Comment:
Would be good to start writing a bit of a decision log about "why rollup" and any other things we're deciding on as part of these initial PRs. If you can add them to the google doc (I've made [one for Rollup here](https://docs.google.com/document/d/1qRgjavvTcbBGEADN2uQxRhqu5fOPQbWV7zzyGullO48/edit#heading=h.e9huuydylpc6) as an example)

Reply:
I think we're still in the decision process so we'll see how people get on with Vite in the meantime.